docs/reference/indices/synced-flush.asciidoc

```diff
@@ -0,0 +1,278 @@
+have identical contents. This means there is no need to copy any segment files
+from the primary to the replica, which saves a good deal of time during
+recoveries and restarts.
+
+This is particularly useful for clusters having lots of indices which are very
+rarely updated, such as with time-based indices. Without the synced flush
+marker, recovery of this kind of cluster would be much slower.
+
+
+[[synced-flush-sync-id-markers]]
+===== Check for `sync_id` markers
+
+To check whether a shard has a `sync_id` marker or not, look for the `commit`
+section of the shard stats returned by the <<indices-stats,indices stats>> API:
+
+[source,console]
+--------------------------------------------------
+GET /twitter/_stats?filter_path=**.commit&level=shards <1>
+--------------------------------------------------
+// TEST[s/^/PUT twitter\nPOST twitter\/_flush\/synced\n/]
+
+<1> `filter_path` is used to reduce the verbosity of the response, but is entirely optional
+
+The API returns the following response:
+
+[source,console-result]
+--------------------------------------------------
+{
+   "indices": {
+      "twitter": {
+         "shards": {
+            "0": [
+               {
+                 "commit" : {
+                   "id" : "3M3zkw2GHMo2Y4h4/KFKCg==",
+                   "generation" : 3,
+                   "user_data" : {
+                     "translog_uuid" : "hnOG3xFcTDeoI_kvvvOdNA",
+                     "history_uuid" : "XP7KDJGiS1a2fHYiFL5TXQ",
+                     "local_checkpoint" : "-1",
+                     "translog_generation" : "2",
+                     "max_seq_no" : "-1",
+                     "sync_id" : "AVvFY-071siAOuFGEO9P", <1>
+                     "max_unsafe_auto_id_timestamp" : "-1",
+                     "min_retained_seq_no" : "0"
+                   },
+                   "num_docs" : 0
+                 }
+               }
+            ]
+         }
+      }
+   }
+}
+--------------------------------------------------
+// TESTRESPONSE[s/"id" : "3M3zkw2GHMo2Y4h4\/KFKCg=="/"id": $body.indices.twitter.shards.0.0.commit.id/]
+// TESTRESPONSE[s/"translog_uuid" : "hnOG3xFcTDeoI_kvvvOdNA"/"translog_uuid": $body.indices.twitter.shards.0.0.commit.user_data.translog_uuid/]
+// TESTRESPONSE[s/"history_uuid" : "XP7KDJGiS1a2fHYiFL5TXQ"/"history_uuid": $body.indices.twitter.shards.0.0.commit.user_data.history_uuid/]
+// TESTRESPONSE[s/"sync_id" : "AVvFY-071siAOuFGEO9P"/"sync_id": $body.indices.twitter.shards.0.0.commit.user_data.sync_id/]
+<1> the `sync id` marker
+
+NOTE: The `sync_id` marker is removed as soon as the shard is flushed again, and
+{es} may trigger an automatic flush of a shard at any time if there are
+unflushed operations in the shard's translog. In practice this means that one
+should consider any indexing operation on an index as having removed its
+`sync_id` markers.
+
+
+[[synced-flush-using-api]]
+===== Use the synced flush API
+
+Use the synced flush API to manually initiate a synced flush.
```

Comment:
Should this be at the beginning of the description section?

Reply:
Addressed with acda611.